web-client/pages/survey.tsx

```diff
@@ -1,54 +1,104 @@
+import {
+    Button,
+    Text,
+    Box,
+    Textarea,
+    Checkbox,
+    Stack,
+    Progress,
+} from '@chakra-ui/react'
 import Head from 'next/head'
-import LinkButton from '../components/LinkButton'
-import { selectConfigState, setConfigState } from '../store/configSlice'
-
-type AnswerType = 'text' | 'checkbox' | 'radio'
+import { setConfigState } from '../store/configSlice'
+import SurveySlider from '../components/SurveySlider'
+import SurveyRadioGroup from '../components/SurveyRadioGroup'
+import axios from 'axios'
+import { useRouter } from 'next/router'
 
 interface Question {
     question: string
     description?: string
-    answerType: AnswerType
 }
 
 const questions: Question[] = [
-    { question: 'With whom are you spending the day?', answerType: 'checkbox' },
+    {
+        question: 'With whom are you spending the day?',
+    },
     {
         question: 'How do you find the weather today?',
-        description:
-            "1= bad and I rather stay at home; 2 = it's okey; 3 = good or even amazing",
-        answerType: 'checkbox',
+        description: '1 - 😒 2 - 🙂 3 - 🤩',
     },
-    { question: 'How many hours did you sleep today?', answerType: 'text' },
+    { question: 'How many hours did you sleep today? 😴' },
     {
-        question: 'How many hours did you spend on work or education?',
-        answerType: 'text',
+        question: 'How many hours did you spend on work? 🧑‍💻',
+    },
+    {
+        question: 'How many hours did you spend on education? 📚',
     },
     {
         question:
             'Thinking about recent days, how satisfied are you with the amount of:',
-        answerType: 'checkbox',
     },
     {
         question: 'Rate your overall health today',
-        answerType: 'checkbox',
+        description: '1 - 🤧 Need to stay in bed, 5 - 🤩 Energized',
     },
     {
         question: 'How would you rate your overall mood today?',
-        answerType: 'checkbox',
+        description:
+            '1 - 😞 Unhappy and discouraged, 5 - 🤩 Happy and inspired',
     },
     {
-        question: 'Any other thoughts you want to record?',
+        question: 'Any other thoughts you want to record? 📖',
         description:
             'If anything that particularly affected your mood happened, you can record it here. Later this information can help you explain certain patterns in your mood curve!',
-        answerType: 'text',
     },
 ]
+
+interface DataModel {
+    name?: string
+    timestamp?: string
+    family: boolean
+    partner: boolean
+    friends: boolean
+    colleagues: boolean
+    weather: number
+    sleep: number
+    work: number
+    study: number
+    sports: number
+    hobbies: number
+    social: number
+    health: number
+    mood: number
+    thoughts: string
+}
+
+const postForm = (data: DataModel): Promise<DataModel> => {
+    console.log('Send request')
+    return axios.post('http://10.100.27.14:8000/api/survey/', {
```

Comment:
do you need the protocol and base of the URL here?  ideally we shouldn't hardcode those

Reply:
ideally lol :DDDDD